src/main/java/HTML1/Task1.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+    <style>
+        table {
+
+            border: 2px solid gray;
+            border-right-color: white;
+            border-left-color: white;
+        }
+
+        h1 {
+            font-size: 150%;
+        }
+
+        th {
+            text-align: left;
+        }
+
+        p {
+            font-family: 'Times New Roman', Times, serif;
+            font-style: italic;
+            font-size: 110%;
+        }
+    </style>
+
+    <title> www.itmathrepetotor.ru </title>
+
+</head>
+<body>
+<h1> Химические элементы </h1>
+<table style="width:22%">
+
+    <tr>
+        <th>Номер</th>
+        <th>
+            Название
+        </th>
+        <th>Символ</th>
+        <th>Год</th>
+        <th>Статья</th>
+    <tr>
+        <td>1</td>
+        <td><a href="url">Водород</a></td>
+        <td><b>H</b></td>
+        <td>1766</td>
+        <td><img style="width:40px; height:25px" src="dz.jpg"></td>
```

Comment:
If you use styles, defined for different selectors at the beginning in `<style>`, then you can also add style for `<img>` since they are similar